### PR TITLE
Add GitHub tiles documentation

### PR DESCRIPTION
**Pull request by Alex-D**

### CODE REVIEW

File: docs/documentation/index.html [hunk 1]
```diff
@@ -829,37 +832,268 @@ <h5 class="m-documentation--configuration-side-title">UI configuration</h5>
...
-             viewBox="0 0 510 510" xmlns="http://www.w3.org/2000/svg">
+        <dt><code>MO_MONITORABLE_GITHUB_COUNTCACHEEXPIRATION</code> <code class="type">number</code>
+        </dt>
+        <dd>
+          Count cache duration in milliseconds. <br>
+          <code><a href="#tile-github-count">GITHUB-COUNT</a></code> tile use
+          <a href="https://developer.github.com/v3/search/#rate-limit">a specific GitHub API with
+          a reduced rate-limit</a>. <br>
+          <p class="note">
+            <span class="tag">Tip</span>
```
Comment (jsdidierlaurent) [inline docs/documentation/index.html]: :+1:

File: docs/documentation/index.html [hunk 2]
```diff
@@ -829,37 +832,268 @@ <h5 class="m-documentation--configuration-side-title">UI configuration</h5>
...
+    "owner": "monitoror",
+    "repository": "monitoror",
+    "ref": "develop"
+  }
+}
+        </code></pre>
+
+        <div class="m-documentation--demo">
+          <div class="m-documentation--demo-tile">
+            <div class="m-documentation--demo-label">monitoror @ develop</div>
```
Comment (jsdidierlaurent) [inline docs/documentation/index.html]: It's 
```monitoror
(branch icon) develop```

File: docs/documentation/index.html [hunk 3]
```diff
@@ -829,37 +832,268 @@ <h5 class="m-documentation--configuration-side-title">UI configuration</h5>
...
+        <dt><code>repository</code> <code class="type">string</code> <span class="required">required</span></dt>
+        <dd>
+          GitHub repository name (from URL)
+        </dd>
+      </dl>
+
+      <div class="m-documentation--example">
+        <pre class="example"><code class="language-json">
+{
+  "type": "GITHUB-CHECKS",
```
Comment (jsdidierlaurent) [inline docs/documentation/index.html]: GITHUB-PULLREQUESTS

File: docs/documentation/index.html (hunk 2 above)
Comment (Alex-D) [reply docs/documentation/index.html]: In a future PR ^^

**Pull request merged**